virt_lightning/api.py

```diff
@@ -142,6 +136,25 @@ def _start_domain(hv, host, context, configuration):
                 network["network"], host["name"], ipaddress.ip_interface(ipv4)
             )
         domain.attach_network(**network)
+
+    if "root_disk_size" in host:
+        logger.warning("The key 'root_disk_size' is deprecated. Use 'disks' instead")
```

Comment:
Can you use a `logger.debug` here instead. We can continue to support for now and wait for the next major release to turn this as an actual warning message.

Reply:
Done.